normits_demand/models/notem/production_models.py

```diff
@@ -766,7 +772,11 @@ def run(self,
             #  Output some audits of what demand was before and after control
             #  By segment.
             if self.constraint_paths is not None:
-                self._logger.info("No code implemented to constrain productions.")
+                msg=(
+                    "No code implemented to constrain productions."
+                )
```

Comment:
Looks like it would all fit on one line - Not that coding standards say there is a soft character line limit of 80 characters, and a hard character line limit of 100 characters. You can set up your IDE to show you where this is